.github/workflows/build.yml

```
@@ -19,4 +19,126 @@
     - name: Build the project
-      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
+      run: dotnet build TestStream.Runner/TestStream.Runner.csproj
+    
+    - name: Trigger Azure DevOps Pipeline
+      env:        
+        AZURE_DEVOPS_ORG: ${{ vars.AZURE_DEVOPS_ORG }}
+        AZURE_DEVOPS_PROJECT: nanoFramework.IoT.TestStream
+        AZURE_DEVOPS_PIPELINE_ID: 111
+        AZURE_POOL_NAME: ${{ vars.AZURE_POOL_NAME }}
+        GITHUB_HEAD_REF: ${{ github.head_ref }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:39:6: Double quote to prevent globbing and word splitting

(shellcheck)